ladybug_comfort/collection/adaptive.py

```diff
@@ -24,21 +24,47 @@
 class Adaptive(ComfortCollection):
     """Adaptive comfort DataCollection object.
 
+    Args:
+        outdoor_temperature: Either one of the following inputs are acceptable:
+
+            *   1 - A Data Collection of prevailing outdoor temperature values in C.
+                Such a Data Collection must align with the operative_temperature
+                input and bear the PrevailingOutdoorTemperature data type in
+                its header.
+            *   2 - A single prevailing outdoor temperature value in C to be used
+                for all of the operative_temperature inputs below.
+            *   3 - A Data Collection of actual outdoor temperatures recorded over
```

Comment:
Since you have the `*` here, I think the numbers are redundant and unnecessary. So we can take them out.